app/src/main/java/org/mozilla/fenix/library/history/HistoryFragment.kt

```diff
@@ -210,23 +210,27 @@ class HistoryFragment : Fragment(), CoroutineScope, BackHandler {
     override fun onBackPressed(): Boolean = (historyComponent.uiView as HistoryUIView).onBackPressed()
 
     private suspend fun reloadData() {
-        val allowedVisitTypes = listOf(VisitType.LINK, VisitType.TYPED, VisitType.BOOKMARK)
+        val excludeTypes = listOf(
+            VisitType.NOT_A_VISIT,
```

Comment:
IIRC you don't need to specify NOT_A_VISIT in the excluded types; it's a meta visit type only used when updating information about the url. Probably doesn't hurt to have it in there, but I'm not 100% sure.